### PR TITLE
Issue/2279 error opening comprehensive in excel (Connect #2279)

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
Broken files were generated when >100 lines were output on the summary sheet for one question.
#### The solution
Do not output frequency table for NUMBER questions, make sure that we never go back up outside the file window (100 lines).
## Checklist
* [x] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

**Comment by stellanl** [comment]
This fixes the two first check boxes of akvo/akvo-product-design#280

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 1]
```diff
@@ -1646,18 +1637,54 @@ private void writeSummaryReport(
...
                         }
                     }
+                    curRow = bottomRow;
 
                     // add a blank row between questions
                     getRow(curRow++, sheet);
                     // flush the sheet so far to disk; we will not go back up
+                    // This is crucial, not just an optimisation!
+                    // File broken if we have too many lines between each flush
                     ((SXSSFSheet) sheet).flushRows(0); // retain 0 last rows and
```
Comment (muloem) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: According to the docs the flushing should happen automatically when accessing `createRow()`.  What i am not sure about is whether the docs are for the more recent version or still valid for the 3.8 that we are using

Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Probably it is just the going back up that is crucial. I can do the experiment.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 2]
```diff
@@ -1471,55 +1473,64 @@ private void writeSummaryReport(
...
                     sheet.addMergedRegion(new CellRangeAddress(curRow - 1, curRow - 1, 0, 2));
                     createCell(
                             row,
                             0,
                             question.getText(),
                             headerStyle);
-                    DescriptiveStats stats = summaryModel
-                            .getDescriptiveStatsForQuestion(
+                    // Variable name
+                    createCell(row, 3,
```
[the remark this answers is not in the document]
Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: It's just a column index,,,

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java (hunk 1 above)
Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Yes, holds for 3.8 also

**Pull request merged**